pay-api/src/pay_api/dtos/product.py

```diff
@@ -0,0 +1,12 @@
+from typing import Optional
+
+from attrs import define
+
+from pay_api.utils.serializable import Serializable
+
+
+@define
+class ProductGetRequest(Serializable):
```

Comment:
We are getting fees and not products, might be better to name it based on the route you are implementing this for.